libpromises/verify_vars.c

```diff
@@ -212,6 +212,11 @@ PromiseResult VerifyVarPromise(EvalContext *ctx, const Promise *pp,
             if (res.status == FNCALL_FAILURE)
             {
                 /* We do not assign variables to failed fn calls */
+                if (EvalContextGetPass(ctx) == 3) {
```

Comment:
Please create a macro for this `3`.

Reply:
We don't want to have to come back and fix this later if convergence passes within a bundle are altered.